.bazelrc

```
@@ -1,3 +1,31 @@
-# Use embedded JDK
-common --repo_env=JAVA_HOME=../bazel_tools/jdk
+common --enable_bzlmod
 common --experimental_google_legacy_api
+
+build --bes_results_url=https://app.buildbuddy.io/invocation/
+build --bes_backend=grpcs://remote.buildbuddy.io
+build --remote_cache=grpcs://remote.buildbuddy.io
+build --experimental_remote_cache_compression
+build --experimental_remote_cache_async
+build --remote_download_toplevel
+build --remote_timeout=3600
+build --build_metadata=REPO_URL=https://github.com/player-ui/rules_player.git
+
+test --bes_results_url=https://app.buildbuddy.io/invocation/
+test --bes_backend=grpcs://remote.buildbuddy.io
+test --remote_cache=grpcs://remote.buildbuddy.io
+test --experimental_remote_cache_compression
+test --experimental_remote_cache_async
+test --remote_download_toplevel
+test --remote_timeout=3600
+test --build_metadata=REPO_URL=https://github.com/player-ui/rules_player.git
+
+coverage --bes_results_url=https://app.buildbuddy.io/invocation/
+coverage --bes_backend=grpcs://remote.buildbuddy.io
+coverage --remote_cache=grpcs://remote.buildbuddy.io
+coverage --experimental_remote_cache_compression
+coverage --experimental_remote_cache_async
+coverage --remote_download_toplevel
+coverage --remote_timeout=3600
+coverage --build_metadata=REPO_URL=https://github.com/player-ui/rules_player.git
```

Can we collapse these into `common`:
```suggestion
common --bes_results_url=https://app.buildbuddy.io/invocation/
common --bes_backend=grpcs://remote.buildbuddy.io
common --remote_cache=grpcs://remote.buildbuddy.io
common --experimental_remote_cache_compression
common --experimental_remote_cache_async
common --remote_download_toplevel
common --remote_timeout=3600
common --build_metadata=REPO_URL=https://github.com/player-ui/rules_player.git
```